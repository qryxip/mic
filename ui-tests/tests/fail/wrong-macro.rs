use mic::solve;

#[solve]
fn main() -> _ {
    42
}
