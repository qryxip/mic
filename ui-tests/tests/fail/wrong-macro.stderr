error[E0277]: `main` has invalid return type `String`
 --> tests/fail/wrong-macro.rs:3:1
  |
3 | #[solve]
  | ^^^^^^^^ `main` can only return types that implement `Termination`
  |
  = help: consider using `()`, or a `Result`
  = note: this error originates in the attribute macro `solve` (in Nightly builds, run with -Z macro-backtrace for more info)
